client/verta/tests/test_dataset_versioning/test_dataset_version.py

```diff
@@ -1,9 +1,11 @@
+# -*- coding: utf-8 -*-
```

Comment:
tl;dr if a dataset version is created with `enable_mdb_versioning=True`:

```python
dataset.create_version(
    Path(filename, enable_mdb_versioning=True),
)
```

then `dataset_version.get_content().download()` is enabled to download that data back from Verta.

If `enable_mdb_versioning=False` (the default), or if a dataset version wasn't created to begin with, then `content.download()` should raise an exception.